### PR TITLE
Conditionally add larger fonts

### PR DESCRIPTION
**Pull request by jcockerill86**
cc @mintuz thoughts on this? Conditionally allowing larger type sizes to be used

Thanks

### CODE REVIEW

File: _typography.scss [hunk 1]
```diff
@@ -18,6 +18,10 @@ $gel-settings-available: false !default;
 $core: true !default;
 $enhanced: true !default;
 
+// If larger font sizes enable, merge them with gel-type-settings
+@if $gel-type-enable--larger-font-sizes {
```
Comment (tomtrumpet) [inline _typography.scss]: The style seems to be using the words type or typography. Could we change references to "font" to fit in with this?

File: lib/_settings.scss [hunk 2]
```diff
@@ -279,6 +284,131 @@ $gel-type-settings: (
...
+        ),
+        'royal-bold': (
+            'group-a': (
+              'font-weight': bold,
+              'letter-spacing': -1px
+            )
+        ),
+        'foolscap': (
+            'group-a': (
+              'font-size': 32px,
```
Comment (tomtrumpet) [inline lib/_settings.scss]: Tabs of 4 or 2?

File: lib/_settings.scss [hunk 3]
```diff
@@ -279,6 +284,131 @@ $gel-type-settings: (
     )
 ) !default;
 
+$gel-larger-font-sizes: (
+    // Additional type sizes for larger contexts
+    //
```
Comment (tomtrumpet) [inline lib/_settings.scss]: Redundant comment line?

File: _typography.scss (hunk 1 above)
Comment (jcockerill86) [reply _typography.scss]: Thanks. Will update to be consistent with naming.

**Comment by JoeHart** [comment]
Looks good to me, merging now!

**Pull request merged**